forc-plugins/forc-client/src/cmd/deploy.rs

```diff
@@ -7,8 +7,13 @@ pub use forc_util::tx_utils::Salt;
 
 use crate::NodeTarget;
 
+forc::cli_examples! {
```

Comment:
Perhaps an example for `--default-signer` , `--testnet` and `--node-url` here would be good also. @kayagokalp do you think we are missing any?

Reply:
Suggesting this for node-url `forc deploy --node-url https://beta-5.fuel.network/graphql`

Reply:
> Perhaps an example for --default-signer , --testnet and --node-url here would be good also. @kayagokalp do you think we are missing any?

With those added I think we are good

Reply:
@cr-fuel I'm happy to merge this PR as is but would you mind opening as issue for adding examples for `--default-signer`, `--testnet` and `--node-url` so we don't forget to get this added.